build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -225,5 +225,12 @@
           DestinationFiles="@(FilesToCopy->'$(StageSymbolsDirectory)\%(RecursiveDir)%(Filename)%(Extension)')" />
 
       <Delete Files="@(PdbsToClean)" />
+
+      <!-- workaround for https://github.com/Microsoft/msbuild/issues/872 -->
+      <Copy SourceFiles="$(RepoRoot)/resources/MSBuild.exe.config"
+            DestinationFiles="$(SdkOutputDirectory)/MSBuild.exe.config" />
```

Comment:
The publishOptions is proving to cause more pain than good. Let's hold off this discussion for the /redist/project.json PR.